.github/workflows/validation-coding-standards.yml

```diff
@@ -0,0 +1,40 @@
+name: 'Validation / Coding Standards'
+
+on: pull_request
+
+jobs:
+  php:
+    runs-on: ubuntu-latest
+    name: PHP
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Composer Install
+        run: composer install --prefer-dist --optimize-autoloader
+
+      - name: PHPCS
+        run: composer lint
+
+  css:
+    runs-on: ubuntu-latest
+    name: CSS
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Yarn Install
+        run: yarn install
```

Comment:
In CI, using `yarn install --immutable` is preferred, as it is faster execution. If we were using NPM, we would run `npm ci` instead.